Gems/Multiplayer/Code/Source/Debug/MultiplayerDebugPerEntityReporter.cpp

```diff
@@ -0,0 +1,389 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "MultiplayerDebugPerEntityReporter.h"
+
+#include <AzCore/Component/TransformBus.h>
+#include <AzCore/Math/ToString.h>
+#include <AzFramework/Entity/EntityDebugDisplayBus.h>
+#include <Multiplayer/IMultiplayer.h>
+
+#if defined(IMGUI_ENABLED)
+#include <imgui/imgui.h>
+#endif
+
+AZ_CVAR(float, net_DebugEntities_ShowAboveKbps, 1.f, nullptr, AZ::ConsoleFunctorFlags::Null,
+    "Prints bandwidth on network entities with higher kpbs than this value");
+
+AZ_CVAR(float, net_DebugEntities_WarnAboveKbps, 10.f, nullptr, AZ::ConsoleFunctorFlags::Null,
+    "Prints bandwidth on network entities with higher kpbs than this value");
+
+AZ_CVAR(AZ::Color, net_DebugEntities_WarningColor, AZ::Colors::Red, nullptr, AZ::ConsoleFunctorFlags::Null,
```

Comment:
Comments are off for the two colours... copy/paste error

Reply:
Good catch, i'll take care of it in a followup.